CASMTRIAGE-3003 - Add step to diagnose and failover MDS (#987)
Cray-HPE/docs-csm: operations/utility_storage/Troubleshoot_Ceph_FS_Client_Connectivity_issues.md

> # Troubleshooting Ceph MDS Client Connectivity Issues

Reviewer: Change "Troubleshooting" to "Troubleshoot"

> Use this procedure to diagnose and fix clients not logging into Ceph fs.

Reviewer: Capitalize "fs"

> **MOTE:** This section does not diagnose nor fix network issues.  Please ensure there that all networking if functional before proceeding.

Reviewer 2: ```suggestion
**MOTE:** This section does not diagnose nor fix network issues.  Please ensure that all networking is functional before proceeding.
```

***IMPORTANT:*** The following commands can be run from ncn-m001/2/3 or ncn-s001/2/3. 

## Procedure

1. Identify if clients are not logged into Ceph FS.

   ```bash

>    # ceph fs status

Reviewer: In general, we are now separating input/output into separate code blocks. For example...

---

```
# ceph fs status
```

Example output:

```
   cephfs - 0 clients    <---- This indicates we have no clients connected
   ======
   RANK      STATE                MDS               ACTIVITY     DNS    INOS
   0        active      cephfs.ncn-s001.abiiiw  Reqs:    0 /s     0      0
   0-s   standby-replay  cephfs.ncn-s002.kyayma  Evts:   38 /s  35.5k  3220
        POOL         TYPE     USED  AVAIL
   cephfs_metadata  metadata  2403M  11.1T
   cephfs_data      data    2641G  11.1T
       STANDBY MDS
   cephfs.ncn-s003.sjatdm
   ```

   cephfs - 0 clients    <---- This indicates we have no clients connected
   ======
   RANK      STATE                MDS               ACTIVITY     DNS    INOS
   0        active      cephfs.ncn-s001.abiiiw  Reqs:    0 /s     0      0
   0-s   standby-replay  cephfs.ncn-s002.kyayma  Evts:   38 /s  35.5k  3220
        POOL         TYPE     USED  AVAIL
   cephfs_metadata  metadata  2403M  11.1T
   cephfs_data      data    2641G  11.1T
       STANDBY MDS
   cephfs.ncn-s003.sjatdm
   ```

1. Fail over the MDS to trigger clients logins.

   ```bash

>    ceph mds fail 0

Reviewer 2: ```suggestion
   # ceph mds fail 0
```

Author: Unless I give a complex example, I tend to not put the # in so they can utilize the copy function of github

   ```

>    NOTE: "0" refers to the active rank in our above output.

Reviewer: Bold "NOTE:"
